terraso_backend/apps/graphql/schema/commons.py

```diff
@@ -43,6 +44,35 @@ def resolve_total_count(self, info, **kwargs):
         queryset = self.iterable
         return queryset.count()
 
+    # This will coax graphene to output more precise types for connections.
+    # Context: https://github.com/graphql-python/graphene/pull/1504
+    # Will be unnecessary after https://github.com/graphql-python/graphene-django/issues/901
+    @classmethod
+    def __init_subclass_with_meta__(cls, node=None, **options):
+        type_name = re.sub("Connection$", "", cls.__name__)
```

Comment:
suggestion: Maybe it would be a good idea to assert the class ends in `Connection`, just in case.

Reply:
it still works if it doesn't, this code provides a cleaner name if it does (and with how our code is structured it always does)